src/server/room-controller.js

```diff
@@ -447,6 +499,7 @@ export default class RoomController {
   registerPages(router) {
     router.get(
       '/rooms/:roomId*',
+      needsAuthentication(),
```

Comment:
Good catch! 😳

Reply:
It replaces the deleted `if (!user) { throw new Unauthorized(); }` :-) (so no change in behavior actually)